### DISCUSSION TITLE
Can't download mods

### DISCUSSION THREAD

**Discussion by Souza-R** [Foxlider/FASTER#104] [Q&A]
Hello guys!

i'm not able to download any mods via FASTER 

"Can't download manifest" 

how do i solve that?

**Comment by ZloyOdessit** [comment]
same problem here
Error: One or more errors occurred. (A task was canceled.) Inner Exception: A task was canceled.
    Download 843577117 completed, it took 0m 12s 65ms

**Comment by therealsputnik** [comment]
Same here, V1.9d
 Error: One or more errors occurred. (A task was canceled.) Inner Exception: A task was canceled.

**Reply by Foxlider** [reply]
1.9d is outdated. 1.9f is the latest release. It contains a fix for the Steam Login issues and possibly more compatibility fixes for SteamAPI

**Comment by priglmeier** [comment]
Here is the workaround from the client side:
Create a new folder named ReforgerMods somewhere.
Go to Steam > Reforger > properties (the gear icon)

Add the launch option to your path like:

-addonDownloadDir "C:\Users\USERNAME\OneDrive\Documents\ReforgerMods"

If you are having issues with server side mods, it can be the mod is no longer compatible, so make certain you are only reviewing the mods on the official Arma Reforger Workshop site. I have had this happen myself.

**Reply by OverlordZorn** [reply]
Reforger?

**Reply by Foxlider** [reply]
Reforger ? 
OneDrive ?

![visible confusion](https://github.com/user-attachments/assets/bbee24c8-61ed-48e0-a824-7763d820cb58)
